iohk-monitoring/src/Cardano/BM/Snocket.hs

```diff
@@ -168,37 +162,18 @@ instance Show (AddressFamily addr) where
 -- 'Socket'.  Snockets are polymorphic over monad which is used, this feature
 -- is useful for testing and/or simulations.
 --
-data Snocket m fd addr = Snocket {
-    getLocalAddr  :: fd -> m addr
-  , getRemoteAddr :: fd -> m addr
-
-  , addrFamily :: addr -> AddressFamily addr
-
-  -- | Open a file descriptor  (socket / namedPipe).  For named pipes this is
-  -- using 'CreateNamedPipe' syscall, for Berkeley sockets 'socket' is used..
-  --
-  , open          :: AddressFamily addr -> m fd
-
-    -- | A way to create 'fd' to pass to 'connect'.  For named pipes it will
-    -- use 'CreateFile' syscall.  For Berkeley sockets this the same as 'open'.
-    --
-    -- For named pipes we need full 'addr' rather than just address family as
-    -- it is for sockets.
-    --
-  , openToConnect :: addr ->  m fd
-
-    -- | `connect` is only needed for Berkeley sockets, for named pipes this is
-    -- no-op.
-    --
-  , connect       :: fd -> addr -> m ()
-  , bind          :: fd -> addr -> m ()
-  , listen        :: fd -> m ()
-
-  , accept        :: fd -> Accept addr fd
-
-  , close         :: fd -> m ()
-
-  }
+data Snocket m fd addr = Snocket
+    { getLocalAddr  :: fd -> m addr
+    , getRemoteAddr :: fd -> m addr
+    , addrFamily    :: addr -> AddressFamily addr
+    , open          :: AddressFamily addr -> m fd
+    , openToConnect :: addr -> m fd
+    , connect       :: fd -> addr -> m ()
+    , bind          :: fd -> addr -> m ()
+    , listen        :: fd -> m ()
+    , accept        :: fd -> Accept addr fd
+    , close         :: fd -> m ()
+    }
```

Comment:
@CodiePP, do we want those comments removed?

Reply:
I think yes, this is OK.

Reply:
found a way how to keep those comments albeit shorter on a single line